packages/docs/docs/intro.md

```diff
@@ -143,6 +143,213 @@ const stake = await membershipMint.getAccountInfo(ixs.output.stakeAccount);
+// they just need enough funds to pay for the transaction fee. If you're using 
+// this code, airdrop a sol to distributionBot.
+
+let distributeToMember1 = await fanoutSdk.distributeWalletMemberInstructions(
+    {
+        distributeForMint: false,
+        member: member1.publicKey,
+        fanout: fanout, // From initialization
+        payer: distributionBot.publicKey,
+    },
+);
+
+const tx = await fanoutSdk.sendInstructions(
+    [...distMember1.instructions],
+    [distributionBot],
+    distributionBot.publicKey
+);
+if (!!tx.RpcResponseAndContext.value.err) {
+    const txdetails = await connection.getConfirmedTransaction(tx.TransactionSignature);
+    console.log(txdetails, tx.RpcResponseAndContext.value.err);
+}
+```
+
+### NFT
+Same as above, but replace distributeToMember1 with this:
+```ts
+const member1.mint = "NFT Mint for Member 1";
+
+let distributeToMember1 = await fanoutSdk.distributeNftMemberInstructions(
+    {
+        distributeForMint: false,
+        member: member1.publicKey,
+        membershipKey: member1.mint,
+        fanout: fanout,
+        payer: distributionBot.publicKey,
+    },
+);
+```
+
+### Token
+
+Same as Wallet, but replace distributeToMember1 with this:
+```ts
+const membershiptMint.publicKey = "SPL-Token-PublicKey";
+
+let distributeToMember1 = await fanoutSdk.distributeTokenMemberInstructions(
+    {
+        distributeForMint: false,
+        membershipMint: membershipMint.publicKey,
+        fanout: fanout,
+        member: member1.publicKey,
+        payer: distributionBot.publicKey,
+    }
+);
+```
+
+### Distribute SPL Tokens
+
+The process is basically the same, you'll additionally specify the Mint of the Token you want to distribute and set distributeForMint to true.
+
+Example for the Wallet member model:
+```ts
+const mint.publicKey = "SPL-Token-To-Distribute-PublicKey";
+
+let distributeToMember1 = await fanoutSdk.distributeWalletMemberInstructions(
+    {
+        distributeForMint: true,
+        member: member1.publicKey,
+        fanout: builtFanout.fanout,
+        payer: distributionBot.publicKey,
+        fanoutMint: mint.publicKey
+    },
+);
+
+```
+
+## Additional Capabilities
+
+One key use case for Hydra is specifying the Hydra Wallet as a creator with some royalty share for an NFT. We've enabled the Authority of the Hydra Wallet to sign NFTs as the Hydra Wallet so the wallet is a verified creator in the NFT metadata. 
+
+import {
+  CreateMasterEditionV3,
+  CreateMetadataV2,
+  Creator,
+  DataV2,
+  MasterEdition,
+  MAX_NAME_LENGTH,
+  Metadata,
+} from "@metaplex-foundation/mpl-token-metadata";
+
+....
+
+const connection = new Connection("devnet", "confirmed");
+let fanoutSdk: FanoutClient;
+
+authorityWallet = Keypair.generate();
+
+fanoutSdk = new FanoutClient(
+            connection,
+            new NodeWallet(new Account(authorityWallet.secretKey))
+        );
+
+const init = await fanoutSdk.initializeFanout({
```

Comment:
@austbot I'm not seeing where the fanout wallet or the authorityWallet sign the metadata for the NFT that's created.